alonet/detr/detr.py

```diff
@@ -31,6 +31,8 @@ def __init__(
         aux_loss=True,
         weights: str = None,
         return_dec_outputs=False,
+        return_enc_outputs=False,
+        return_bb_outputs=False,
```

Comment:
For consistency between detr & deformable, we have to make sure theses will also be include in deformable. (pull master before to do it sine there some change that might be relevant for you)

Reply:
Done